liiatools_pipeline/sensors/job_success_sensor.py

```diff
@@ -72,22 +67,24 @@ def move_current_sensor(context):
     default_status=DefaultSensorStatus.RUNNING,
 )
 def concatenate_sensor(context):
```

Comment:
This looks like maybe it would be good to run black formatting against it as there should be empty lines between functions.

Reply:
I just ran black formatting and it didn't change anything. Also not sure I understand where you're referring to exactly as on my end it looks like there are two empty lines between functions

Reply:
Nevermind. Sick brain strikes again